### ISSUE TITLE
TabIndex issue

### ISSUE THREAD

**Issue by Andree643** [jquery/jquery-mobile#8078]
When I try to exclude the form element from tab flow, the tabindex attribute is ignored.
See the example bellow.

http://codepen.io/Andree643/pen/raXdEb

**Comment by gabrielschulhof** [comment]
@Andree643 Can you please reproduce this problem with our latest code? http://jsbin.com/huvoraba/1/edit provides a great starting point.

**Comment by Andree643** [comment]
Here is the problem on jsBin

http://jsbin.com/mirupigade/2/

**Comment by arschmitz** [comment]
The problem is we are not copying the tab index from the native select. To the new select.

**Comment by Andree643** [comment]
I can see that. Isn't this possible to implement this feature? In my opinion this is quite useful since you can use forward/backward tabbing among the fields in eg. Apple devices.

**Comment by arschmitz** [comment]
@Andree643 yes it is and yes it should be there, but we have to identify and confirm the cause of an issue before we can fix it.

**Comment by Andree643** [comment]
@arschmitz So this feature should be already present, but there is a bug if I understand it correctly.
